python/src/LatentVariableModel_doc.i.in

```diff
@@ -0,0 +1,144 @@
+ranging from :math:`0` to :math:`l-1`. However, this representation is purely practical, and the values
+assigned to each level have no practical meaning or effect.
+
+Finally, for a similar reason, when using this type of kernel when defining a Gaussian process, it is suggested to 
+rely on a constant functional basis.
+
+See Also
+--------
+CovarianceModel
+
+Examples
+--------
+Create a latent model covariance function with a latent space of dimension 2, for a discrete variable 
+characterized by 3 levels:
+
+>>> import openturns.experimental as otexp
+>>> covModel = otexp.LatentVariableModel(3, 2)
+>>> activeCoordinates = [0.1, 0.3, -0.4]
+>>> covModel.setLatentVariables(activeCoordinates)
+>>> print(covModel(1, 2))
+[[ 0.904837 ]]
+>>> print(covModel(0, 2))
+[[ 0.882497 ]]
+>>> print(covModel(1, 1))
+[[ 1 ]]"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::computeAsScalar
+"Compute the covariance function for scalar model.
+
+Available usages:
+    computeAsScalar(s, t)
+
+Parameters
+----------
+s, t : floats, must have integer values between :math:`0` and :math:`l-1`
+
+Returns
+-------
+covariance : float
+    Covariance.
+
+Notes
+-----
+The method makes sense only if the dimension of the process is :math:`d=1`.
+It evaluates :math:`C(s, t)`."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::getActiveLatentVariables
+"Active latent variables accessor.
+
+Parameters
+----------
+activeLatentVariables : :class:`~openturns.Point`
+    Active coordinates of the discrete levels in the latent space. 
+    The inactive coordinates are set to 0 (i.e., the first latent 
+    variable is placed in the Euclidean space origin, and the second
+    latent variable can only be placec onto the first axis)."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::getFullLatentVariables
+"Latent variables accessor.
+
+Returns
+-------
+activeLatentVariables : :class:`~openturns.Point`
+    Coordinates of the discrete levels in the latent space."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::getLatentDimension
+"Latent dimension accessor.
+
+Returns
+-------
+LatentDimension : int
+    Dimension of the latent space."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::getNLevels
+"Number of levels accessor.
+
+Returns
+-------
+NLevels : int
+    Number of levels characterizing the discrete variable."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::setLatentVariables
+"Number of levels accessor.
+
+Parameters
+----------
+LatentVariables : :class:`~openturns.Point`
```

Comment:
- idem on the upper case first letter
- SWIG automatically converts any sequence of floats into a `Point`.

This is why the sented `sequence of float` is consistently used in the doc (e.g. in the doc of [Sample.add()](https://github.com/openturns/openturns/blob/650bb5e463f62d5e5519615655e33db87d2ca52c/python/src/Sample_doc.i.in#L210)).